### PR TITLE
Combine support for ApexyLoader

### LINKED ISSUE #29: Make Apexy Loader more reactive
Update API of Apexy Loaders for using it in Rx style.
Or maybe add an extensions that wraps Loader into Combine Publisher.

### PR DESCRIPTION
**Pull request by subdan**
Add ability to observe loader state via Combine.

Closes #29

### CODE REVIEW

File: Sources/ApexyLoader/ContentLoader.swift [hunk 1]
```diff
@@ -28,8 +29,18 @@ open class ContentLoader<Content>: ObservableLoader {
...
+            }
         }
     }
+    
+    /// Content loading status. The default value is `.initial`.
+    ///
+    /// - Remark: To change state use `update(_:)`.
+    @available(iOS 13.0, *)
+    @available(macOS 10.15, *)
+    private(set) public lazy var statePublisher = CurrentValueSubject<LoadingState<Content>, Never>(.initial)
```
Comment (Alexander-Ignition) [inline Sources/ApexyLoader/ContentLoader.swift]: library users can complete the publisher
https://developer.apple.com/documentation/combine/currentvaluesubject/send(completion:)

it seems that this is an unexpected behavior

File: Sources/ApexyLoader/ContentLoader.swift [hunk 2]
```diff
@@ -1,3 +1,4 @@
+import Combine
```
Comment (modestman) [inline Sources/ApexyLoader/ContentLoader.swift:2]: Do you need to add `#if canImport(Combine)`?

File: Sources/ApexyLoader/ContentLoader.swift (hunk 1 above)
Comment (subdan) [reply Sources/ApexyLoader/ContentLoader.swift]: Fixed. Please check it out.

File: Sources/ApexyLoader/ContentLoader.swift (hunk 2 above)
Comment (subdan) [reply Sources/ApexyLoader/ContentLoader.swift:2]: Added.

**Review by modestman — APPROVED** [review]

**Review by Alexander-Ignition — APPROVED** [review]

**Pull request merged**